subprojects/core/src/test/java/hu/bme/mit/theta/core/utils/ExprSimplifierTest.java

```diff
@@ -269,6 +270,7 @@ public void testIntAdd() {
 		assertEquals(Add(a, Int(4)), simplify(Add(Int(1), Add(a, Int(3)))));
 		assertEquals(a, simplify(Add(Int(-3), a, Int(3))));
 		assertEquals(Add(a, b, a, b, c), simplify(Add(a, Add(b, Add(a, Add(b, c))))));
+		assertEquals(Int(new BigInteger("4294967294")), simplify(Add(Int(Integer.MAX_VALUE), Int(Integer.MAX_VALUE))));
```

Comment:
Nice

Reply:
Right now `Int` takes either `int` or a `BigInteger`. What do you think, should I implement az overload that takes a `String`, making it easier to create "big" literals?

Reply:
Yes, I think that would be useful. Basically that would just pass `new BigInteger(string)` to the existing constructor, right?

Reply:
By constructor I mean the static factory method.

Reply:
Implemented in b05cd37